modules/ztp-gitops-ztp-max-spoke-clusters.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc]
+
+:_content-type: REFERENCE
+[id="ztp-gitops-ztp-max-spoke-clusters_{context}"]
+= Recommended hub cluster specifications and managed cluster limits for {ztp}
+
+With {ztp-first}, you can manage thousands of clusters in geographically dispersed regions and networks.
+The Red Hat Performance and Scale lab successfully created and managed 3500 virtual {sno} clusters with a reduced DU profile from a single {rh-rhacm-first} hub cluster in a lab environment.footnote:[link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/install/installing#single-node[Creating and managing {sno} clusters with {rh-rhacm}]]
```

Comment:
The repo guidelines say to avoid footnotes: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#footnotes

I think you could add this to the additional resources section